### PR TITLE
Property Widget 1.0

### PR DESCRIPTION
**Pull request by saskliutas**

### CODE REVIEW

File: packages/itwin/property-grid/package.json [hunk 1]
```diff
@@ -29,28 +29,39 @@
     "copy:public": "cpx \"./public/**/*\" ./lib/public",
     "copy:cjs": "cpx \"./src/**/*.scss\" ./lib/cjs",
     "copy:esm": "cpx \"./src/**/*.scss\" ./lib/esm",
-    "cover": "",
+    "cover": "nyc npm run test",
```
Comment (aruniverse) [inline packages/itwin/property-grid/package.json:32]: can you update the repository field, can look at how its done in the core repo now

File: packages/itwin/property-grid/package.json [hunk 2]
```diff
@@ -29,28 +29,39 @@
...
     "lint:fix": "npm run lint -- --fix",
     "pseudolocalize": "betools pseudolocalize --englishDir ./public/locales/en --out ./lib/public/locales/en-PSEUDO",
-    "test": "",
+    "test": "mocha \"./lib/cjs/test/**/*.test.js\"",
     "clean": "rimraf lib .rush/temp/package-deps*.json",
     "rebuild": "npm run clean && npm run build"
   },
+  "peerDependencies": {
+    "@itwin/appui-abstract": "^4.0.0",
+    "@itwin/appui-react": "^4.0.0",
```
Comment (aruniverse) [inline packages/itwin/property-grid/package.json:43]: should appui peers be ^4.1 to match the devDeps?

**Review by aruniverse — APPROVED** [review]

**Comment by aruniverse** [comment]
1 last thing, mind updating the root readme? moving up property-grid to 4.x compatible

File: packages/itwin/property-grid/package.json (hunk 2 above)
Comment (saskliutas) [reply packages/itwin/property-grid/package.json:44]: No there is no need to bump peerDeps at the moment.

**Pull request merged**